### ISSUE TITLE
Sticker-related in Audit Log Events

### ISSUE THREAD

**Issue by book000** [DV8FromTheWorld/JDA#2196]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [X] I have checked the branches or the maintainers' PRs for upcoming features.

### Feature Request

Thank you for developing this wonderful library.

I would like to request implementation of Sticker related `ActionType` in Audit Logs.  
According to the Discord Developer Portal, the following Sticker-related events are available in Audit Logs Events ([reference](https://discord.com/developers/docs/resources/audit-log#audit-log-entry-object-audit-log-events)):

- `STICKER_CREATE`
- `STICKER_UPDATE`
- `STICKER_DELETE`

These events do not appear to be implemented in JDA `v5.0.0-alpha.17`: [net.dv8tion.jda.api.audit.ActionType](https://github.com/DV8FromTheWorld/JDA/blob/master/src/main/java/net/dv8tion/jda/api/audit/ActionType.java)  
And, no Issue or PR related to this issue was found: [Search `sticker audit` in Issue/PR](https://github.com/DV8FromTheWorld/JDA/issues?q=sticker+audit)

With these implemented, when a Sticker is changed, it will be possible to retrieve information from the audit log about the user who changed it.

### Example Use-Case

```java
/**
 * Gets the user who recently changed the Sticker.
 *
 * @param sticker Target guild sticker
 * @return User who changed the Sticker, or null if not found.
 */
User getChangedStickerUser(GuildSticker sticker) {
    if (sticker.getGuild() == null) {
        throw new IllegalArgumentException("sticker.getGuild() is null");
    }
    List<AuditLogEntry> entries = sticker
        .getGuild()
        .retrieveAuditLogs()
        .type(ActionType.STICKER_UPDATE)
        .limit(5)
        .complete();
    if (entries.isEmpty()) {
        return null;
    }
    return entries
        .stream()
        .filter(entry -> entry.getTargetIdLong() == sticker.getIdLong())
        .map(AuditLogEntry::getUser)
        .filter(Objects::nonNull)
        .findFirst()
        .orElse(null);
}
```

**Comment by book000** [comment]
**For those who have seen this Issue**: This issue has been addressed in #2198. It will be resolved in the next version.

**Comment by MinnDevelopment** [comment]
Available in [alpha 18](https://github.com/DV8FromTheWorld/JDA/releases/tag/v5.0.0-alpha.18)
